addons/l10n_uy/tests/test_check_vat.py

```diff
@@ -0,0 +1,71 @@
+from odoo.exceptions import ValidationError
+from odoo.tests import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class CheckUyVat(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='uy'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+    @classmethod
+    def _create_partner(cls, identification_type, vat):
+        return cls.env['res.partner'].create({
+            'name': 'Uruguayan Partner',
+            'l10n_latam_identification_type_id': cls.env.ref(f'l10n_uy.{identification_type}').id,
+            'vat': vat,
+            'country_id': cls.env.ref('base.uy').id
+        })
+
+    def test_valid_ci(self):
+        # Valid CI
+        partner = self._create_partner('it_ci', '3:402.010-1')
+        self.assertEqual(partner._l10n_uy_ci_nie_is_valid(), True)
```

Comment:
Change all `self.assertEqual(..., True)` to `self.assertTrue(...)`
```suggestion
        self.assertTrue(partner._l10n_uy_ci_nie_is_valid())